### PR TITLE
Balance quarterstaff and sling staff

### PR DESCRIPTION
**Pull request by Salty-Panda**
#### Summary
Balance "Balance quarterstaff and sling staff"

#### Purpose of change

The quarterstaff is just a whittled-down long branch, it's not treated nor reinforced in any way so it shouldn't have the DURABLE flag.
Sling staff is just an identical whittled down long branch but with a string attached and has very different stats.

#### Describe the solution

Check sources:
https://qtimber.daf.qld.gov.au/guides/wood-density-and-hardness,
https://www.quora.com/How-much-does-a-quarterstaff-weigh, https://www.woodenswords.com/Quarterstaff_p/staff.round.1.25x92.htm, https://www.woodenswords.com/Laminated_Hickory_Stave_p/staff.octagon.1-1-16x72.htm, https://cedarstripkayak.wordpress.com/lumber-selection/162-2/,
https://extension.psu.edu/calculating-the-green-weight-of-wood-species#section-1

Professional staffs with length ~180cm made from dried oak/hickory weight around 1kg, which translates to 1250ml.
Assuming the player uses fresh wet oak which is 1g/ml I adjusted current parameters. I decided on the volume of 1500ml as a baseline (1,6 cm radius, 180cm long, rather common dimensions).

Reduce discrepancies between both, remove the durable flag.
Add sling staff to martial arts supporting quarterstaff.
Left sling staff as a subpar melee weapon in order to keep them balanced gameplay-wise, although looking at pics the difference is negligible.

Update tests (once I get results from website)
Previous tests results: 17 for normal, 18 for ironshod, 17 for powered
Current tests results: 20.8 for normal, 22.8 for ironshod, 21.7 for powered
Added test for sling staff

#### Describe alternatives you've considered

Making a new "improvised quarterstaff", making staff sling nearly identical to a quarterstaff, making a new quarterstaff requiring a treated pole and leather to craft.

#### Testing
none, but I assume that higher DPS will be outweighed by increased stamina consumption, since weights increased

### CODE REVIEW

**Comment by Salty-Panda** [comment]
It breaks the dps test for staves, imma update it sometime soon

**Comment by I-am-Erk** [comment]
I think your "improvised quarterstaff" or just "wooden staff" would be a better option than removing durable_melee from all quarterstaves. Quarterstaves are excellent weapons, the problem here isn't with the quarterstaff item but with the ease of crafting. A proper quarterstaff would need to be seasoned, but we can at least sort of represent that with a slightly less accessible recipe as you've described. One of these days we'll have the ability to season wood...

I think I would keep the sling-staff's to-hit at 1, rather than 2, since it's presumably a bit more unwieldy. I agree it should not have the durable_melee flag since there's a piece on it that can get ripped off that is not a problem for a quarterstaff.

**Comment by Salty-Panda** [comment]
I did it in this way mostly because of current Freezes. Based on the recipe I assumed that the staff you can craft early game in less than an hour is an improvised item, but I didn't change the strings.

So should I cut crop this PR or prepare a follow-up PR that could be merged after freezes (that changes name and adds stronger version that is harder to craft and requires heat source) to show that the issue won't be abandoned in a half-done state?

File: data/json/items/ranged/slings.json [hunk 1]
```diff
@@ -86,13 +85,13 @@
...
     "skill": "throw",
     "ammo": [ "rock" ],
-    "weight": "2000 g",
-    "volume": "3 L",
+    "weight": "1600 g",
+    "volume": "1650 ml",
     "longest_side": "180 cm",
     "price_postapoc": 250,
-    "to_hit": 1,
+    "to_hit": 2,
```
Comment (Salty-Panda) [inline data/json/items/ranged/slings.json:92]:
```suggestion
    "to_hit": 1,
```
Revert as per Erk's suggestion

**Comment by I-am-Erk** [comment]
It's unlikely we'll merge for stable, I'd just do it all in this PR. Stable should be done in a few hours, I'm not merging much else.

**Pull request merged**